examples/lb/listener/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "vkcs_lb_listener" "listener" {
+  name            = "listener"
```

Comment:
give more meaning names, also add description where it is possible

Reply:
changed